.github/workflows/test_pr_changes.yml

```diff
@@ -45,6 +47,12 @@ jobs:
           filters: |
             wire-ios:
               - 'wire-ios/**'
+            wire-reusable-ui-components:
+              - 'WireUI/Sources/WireReusableUIComponents/**'
+              - 'WireUI/Tests/WireReusableUIComponentsTests/**'
+            wire-design:
+              - 'WireUI/Sources/WireDesign/**'
+              - 'WireUI/Tests/WireDesignTests/**'
```

Comment:
I think we should actually stop testing individual targets only.
For example: changes in WireDesign directly affect Wire-iOS, so I might change these lines into:
```suggestion
            wire-ios:
              - 'wire-ios/**'
              - 'WireUI/**'
```
Please share your opinions.